tests/codecs/test_iot_ext_codec.py

```diff
@@ -35,7 +35,7 @@
 
 class IoTextCodecTest(TestCase):
 
-    # TODO: verify where is a bug!?
+
     @skip
```

Comment:
I think you can remove this skip safely now.

Reply:
oh I fixed only second test, let me take a look at the skipped one. I will reply a bit later.

Reply:
Then I think we should keep the comment

Reply:
The explanation of the issue with test_decode tests case has been described in the issue #1, you can take a look.

Reply:
second test also fixed.